### ISSUE TITLE
r10 connection

### ISSUE THREAD

**Issue by galway1984** [travislang/gspro-garmin-connect-v2#12]
any advice appreciated.
running garmin connect v2 it connects to the gspro straight away. however when i connect via garmin app it connects but immediately disconnects. i have turned on/off firewall.
the issue is on my end as i have tried mholowsko/gsp-r10-adapter and the exact same issue it connects but immediately disconnects. just very little trouble shooting guidance out there and if anyone could help i would be very grateful. thanks Happy New Year

**Comment by travislang** [comment]
what version of this program are you using?

**Comment by galway1984** [comment]
See link below for example
https://photos.app.goo.gl/QmUvj7MuduktDT129

Thanks

**Comment by travislang** [comment]
Can you try downloading the old version (v1.2.1) and try that one.  Let me know if you experience the same issue with the old version

**Comment by that0n3guy** [comment]
I'm getting disconnects like crazy now too.  It's been working perfect for months, today I couldn't hit three balls without an issue.

I'm on Android with Garmin app version 2.10.1.   I'm guessing there's an update that makes the Garmin app more picky.

**Comment by Kieffer87** [comment]
> @that0n3guy what version of this app?
> 
> I don’t have an R10 anymore so just trying to verify that the update I pushed like a week ago isn’t the cause. 
> 
> Also would be curious if anyone is having issues on the iOS app

No problems on my end running the latest version. On Android though.

**Comment by jonweiner** [comment]
Took me a few days to realize it, but it's the windows defender firewall that is blocking the API connection to GS pro (in my case). Hope this helps.

**Comment by jonweiner** [comment]
Or, the same thing was happening to me and this is how I resolved the problem.

**Comment by galway1984** [comment]
I finally got mine just going. Friend who works with IT ran the programme with Windows 8 instead of win 10. Worked straight awayThanks for advice earlierOn 14 Jan 2023 14:51, Todd Kieffer ***@***.***> wrote:



@that0n3guy what version of this app?

I don’t have an R10 anymore so just trying to verify that the update I pushed like a week ago isn’t the cause.

Also would be curious if anyone is having issues on the iOS app



No problems on my end running the latest version. On Android though.
